pages/domains/[name]/index.tsx

```diff
@@ -59,6 +63,12 @@ export function ManageNamePageContent({ name }: { name: string }) {
     handleFetchENSDomainData();
   }, []);
 
+  useEffect(() => {
```

Comment:
You can move the "handleFetchENSDomainData()", to the "EditModalContent->closeModal" and avoid the usage of the hook useEffect

Reply:
@lgahdlIf I move handleFetchENSDomainData() to be called within the closeModal function, the page content will re-render every time the modal closes, even if there were no changes made to the data

Reply:
Oh, I see, the updateFieldsWithEnsData function changes the layout of the fields, you're right. Ok then

Reply:
@lgahdl Actually, you are right. useEffect will also rerender the data every time the modal closes. I updated the component so new info is only fetched  when necessary